### ISSUE TITLE
AllowedIPs option that has more priority than AllowedApps

### ISSUE THREAD

**Issue by EntityinArray** [wiresock/WireSockUI#90]
If you use the `AllowedApps` together with `AllowedIPs = 0.0.0.0/0`, you lose ability to contact your Wireguard peers from other apps.
Could we add something like `ForceAllowedIPs = 10.6.6.0/24`, so that you can both limit VPN on per-app basis, but have Wireguard peers connectivity from all apps?

`ForceAllowedIPs` is a stupid name, maybe someone can come up with something better. But we really need this, because right now I don't see any way to achieve this behavior with WireSocks

Basically, i want to connect to wireguard peer IPs, regardless of whether app is in AllowedApps.

**Comment by wiresock** [comment]
This request has been moved to a GitHub Discussion for product/design discussion before any implementation work is scoped: https://github.com/wiresock/WireSockUI/discussions/160
